ribbon-loadbalancer/src/main/java/com/netflix/loadbalancer/BaseLoadBalancer.java

```diff
@@ -827,15 +811,12 @@ public void forceQuickPing() {
         if (canSkipPing()) {
             return;
         }
-        if (logger.isDebugEnabled()) {
-            logger.debug("LoadBalancer:  forceQuickPing invoked");
-        }
-        Pinger ping = new Pinger(pingStrategy);
+        logger.debug("LoadBalancer [{}]:  forceQuickPing invoking", name);

```

Comment:
Are we pinging connections before sending requests? (not something related to this pull request, but just something I'm interested in learning)